images/macos/provision/utils/xamarin-utils.sh

```diff
@@ -19,7 +19,11 @@ buildVSMacDownloadUrl() {
 }
 
 buildMonoDownloadUrl() {
-  echo "https://dl.xamarin.com/MonoFrameworkMDK/Macx86/MonoFramework-MDK-${1}.macos10.xamarin.universal.pkg"
+if [[ $1 -eq 6.12.0.182 ]]; then 
```

Comment:
Why do we need this condition if we don't pass that version?

Reply:
We'll need it after the announcement, but if it shouldn't be here before the announcement, I can add it later
@ddobranic, what do you think about this?

Reply:
we don't need announcement for Mono, comment related to announcement was related to default version of xcode. see bellow https://github.com/actions/virtual-environments/pull/5764/files#r901084818